docs/package.json

```
@@ -6,19 +6,23 @@
     "docusaurus": "docusaurus",
     "start": "docusaurus start",
     "build": "docusaurus build",
+    "build-all": "npm run build && npm run gen-api-docs",
```

```suggestion
    "build-all": "npm run gen-api-docs && npm run build",
```

This should be inverted